Source/ACE/Entity/Creature.cs

```diff
@@ -698,5 +698,54 @@ public override void ActOnUse(ObjectGuid playerId)
                 player.Session.Network.EnqueueSend(sendUseDoneEvent);
             }
         }
+
+        public void HandleActionMotion(UniversalMotion motion)
```

Comment:
Also, these probably belong in WorldObject --- I believe any object can have a motion, not just creatures.